libsplinter/src/biome/key_management/database/postgres/migrations/2019-12-12-200956_create-keys/up.sql

```diff
@@ -18,5 +18,6 @@ CREATE TABLE IF NOT EXISTS keys (
     encrypted_private_key TEXT NOT NULL,
     user_id               TEXT NOT NULL,
     display_name          TEXT NOT NULL,
-    PRIMARY KEY(public_key, user_id)
```

Comment:
As this migration may be part of a release, this change should probably be in a separate migration.

Reply:
Addressed